app/presenters/photo_presenter.rb

```diff
@@ -10,7 +10,8 @@ def base_hash
         small: url(:thumb_small),
         medium: url(:thumb_medium),
         large: url(:scaled_full)
-      }
+      },
+      location: location,
```

Comment:
- Align the elements of a hash literal if they span more than one line.
- Avoid comma after the last item of a hash.